GraphQL-TSC.md

```diff
@@ -162,24 +168,43 @@ When something cannot be decided by consensus or a formal vote is required, the
 
 The ballots may be public or private depending upon the situation, but the aggregate results should always be published.
 
-#### Minimum threshold for votes
+### Defining a quorum of "attending" members
+
+To balance preserving the voting ability of all TSC members with the desire for the voting process to remain efficient and avoid deadlocks, we define a valid quorum based on members' recent attendance of GraphQL Working Group meetings.
 
-Because we work in a distributed environment, the voting process must account for a range of time zones and schedules. A quorum of at least half (or supermajority when relevent) of the TSC voting members must cast a ballot in order for the vote to be valid. Once the voting threshold has been met, one of these two critera must be satisfied to conclude the vote:
+A quorum is a majority (more than half, or 2/3 for a supermajority vote) of the TSC *attending members*. A quorum must cast a ballot in order for a vote to be valid.
+
+A TSC *attending member* is a member who has attended one of the previous three meetings. Should a TSC member miss three consecutive meetings, they will no longer be counted when determining quorum (but may still vote). A member starts counting towards quorum as of attending a meeting.
+
+### Voting process
+
+Because we work in a distributed environment, the voting process must account for a range of time zones and schedules. Once the threshold of a quorum has been met and a vote is valid, one of these two critera must be satisfied to conclude a vote:
 
 * A notice is sent via email that the vote will conclude in three business days, reminding those who haven't voted that they should do so. The vote will conclude at the end of this time.
-* The election results will not change if additional TSC members vote.
+* The election results would not change with the addition of another vote.
 
-#### Ensuring efficient votes
+Once a valid vote is concluded, the result is determined by the number of votes recieved at that time (as opposed to the total number of TSC members):
 
-A TSC member must attend two of the previous five meetings to be considered a "TSC voting member" and be eligible to vote. Should a TSC member miss three or more of the prior five meetings, they will no longer vote or be counted toward the total number of voting members. This is to ensure broad participation as well as to avoid voting deadlocks.
+* For a simple majority (or supermajority), the votes in favor must exceed half (or 2/3) of the total number of votes.
+* For the Condorcet method, all votes received at the time the vote is concluded are considered.
 
-A member may regain their voting status by attending two meetings and may participate in votes in the second meeting.
+TSC members are not required to vote. An abstention vote is not considered part of the total set of votes, nor towards a voting quorum.
```

Comment:
Hmmm not sure this makes sense. Usually an explicit vote of abstention counts towards quorum (you’re saying “I was present, understood the argument, and do not care which way the result goes”) whereas an abstention via not voting doesn’t count towards quorum (as that’s saying you may not have had time to figure out what makes sense). We may want to have both a “present” and “abstain-do-not-understand-yet” vote.

Reply:
I think my worry is that if an abstain counts as a vote but we consider a passing result for majority based on yesVotes/allVotes then an abstain is equivalent to a no vote.

I think the only way you can make an abstain not affect the outcome is by also removing it from the denominator.

But if you had a bunch of people abstain from voting, would you still want to consider those towards meeting quorum for considering a vote valid? I suppose that's what a "present" vote would imply? Do we want to support that or should we require yes/no votes for measures to pass?

Reply:
I would be OK with any of:
- an explicit abstention being impossible (i.e. the vote continues until quorum is reached and the deadline to keep voting passes)
- abstentions counting towards quorum but reducing the vote’s denominator for majority required
- abstention reducing the denominator without adding to the numerator both for quorum and also majority

An abstention not counting towards quorum is equivalent to the first, except it makes it more explicit that if you say you’re abstaining, your vote might still actually be required to close out the issue. Especially if there were 2 attending TSC members, you don’t want one of them to feel like it’s OK to abstain if their vote is in fact required to come to a decision: both of them really do need to keep digging into the issue until they have an explicit opinion, or we need to let one of them completely defer the option to the other to enable a quorum from both with a 1-vote majority.

Reply:
Hmm taking those three examples out, if we had 3 attending members, then:
- in the first, you always need 2 to vote up or down for there to be quorum. If one votes yes and one no, it fails.
- In the second, you can have one person vote yes and one abstain. Quorum is achieved (2/3), and it will fail (1/2). Whereas if you have two people abstain and one vote yes, it will have quorum (3/3) and pass (1/1). 
- in the third, one person abstaining with one yes vote is not enough to reach quorum (1/2), while two abstaining and one voting achieves quorum (1/1) and still lets the one vote pass it (1/1).